shell/platform/windows/flutter_windows_view.cc

```diff
@@ -255,7 +287,27 @@ bool FlutterWindowsView::ClearContext() {
 }
 
 bool FlutterWindowsView::SwapBuffers() {
-  return surface_manager_->SwapBuffers();
+  // raster task runner
```

Comment:
I wouldn't necessarily name the specific thread for the raster thread cases since that's up to the engine; instead you could say something like "Called on an engine-controlled (non-platform) thread."

(It's particularly confusing to mention in an embedding built on the API, because the API never mentions the raster thread.)